content/src/roadmaps/industries/certified-public-accountant.json

```diff
@@ -51,6 +51,12 @@
       "licenseTask": "register-accounting-firm",
       "required": false,
       "weight": 9
+    },
+    {
+      "step": 4,
+      "licenseTask": "public-accountant-license",
+      "required": false,
+      "weight": 10
```

Comment:
[boulder] This doesn't need to be here. The question is a part of the CPA add-on which is in `cpa.json`. There's an onboarding question `isCpaRequiredApplicable` that controls whether or not the add-on is applied. Adding to the roadmap will result in this task being added twice for the accounting industry.

Reply:
I was going to say the same thing